server/models/Pet.js

```diff
@@ -0,0 +1,51 @@
+import pg from "pg"
+
+const pool = new pg.Pool({ connectionString: "postgres://postgres:password@localhost:5432/new_family_member_development" })
+
+class Pet {
+    constructor({ name, img_url, imgUrl, age, vaccination_status, vaccinationStatus,adoption_story, adoptionStory, available_for_adoption,availableForAdoption, pet_type_id, petTypeId }) {
+        this.name = name
+        this.imgUrl = imgUrl || img_url
+        this.age = age
+        this.vaccinationStatus = vaccinationStatus || vaccination_status
+        this.adoptionStory = adoptionStory || adoption_story
+        this.availableForAdoption = availableForAdoption || available_for_adoption
+        this.petTypeId = petTypeId || pet_type_id
+    }
+
+    // static async findAll() {
+    //     try {
+    //       const result = await pool.query("SELECT * FROM adoptable_pets;")
+    //       const petsData = result.rows
+    //       const pets = petsData.map(pet => new this(pet))
+
+    //       return pets
+
+    //     } catch (error) {
+    //         console.log(error)
+    //         throw (error)
+    //     }
+    // }
```

Comment:
Not sure if these methods were meant to be commented out? Typically we don't want to merge incomplete/ unused code